docs/UserGuide.md

```diff
@@ -299,7 +329,8 @@ Action | Format, Examples
 **Search** | `search KEYWORD [MORE_KEYWORDS]`<br> e.g., `search James Jake`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Clear** | `clear`
-**Archive** |  1. **Archive \(1 entry\):**  `archive INDEX`<br> e.g., `archive 1` <br>2. **Archive \(all\):**  `archvie all` <br>3. **List all archived data:**  `archvie list`</br>
+**Archive** |  1. **Archive \(1 employee\):**  `c-archive INDEX`<br> e.g., `c-archive 1` <br>2. **Archive \(all emp;oees\):**  `c-archvie all` <br>3. **List all archived data:**  `c-archvie list`</br>
```

Comment:
**Archive \(all emp;oees\):** here should be employees?

Reply:
Thank you! Have corrected this part.